contrib/linuxarm64v8.Dockerfile

```diff
@@ -0,0 +1,114 @@
+# This dockerfile is meant to cross compile with a x64 machine for a arm32v7 host
+# It is using multi stage build: 
+# * downloader: Download litecoin/bitcoin and qemu binaries needed for c-lightning
+# * builder: Cross compile c-lightning dependencies, then c-lightning itself with static linking
+# * final: Copy the binaries required at runtime
+# The resulting image uploaded to dockerhub will only contain what is needed for runtime.
+# From the root of the repository, run "docker build -t yourimage:yourtag -f contrib/linuxarm32v7.Dockerfile ."
+FROM debian:stretch-slim as downloader
+
+RUN set -ex \
+	&& apt-get update \
+	&& apt-get install -qq --no-install-recommends ca-certificates dirmngr wget \
+     qemu qemu-user-static qemu-user binfmt-support
+
+WORKDIR /opt
+
+RUN wget -qO /opt/tini "https://github.com/krallin/tini/releases/download/v0.18.0/tini-arm64" \
+    && echo "7c5463f55393985ee22357d976758aaaecd08defb3c5294d353732018169b019 /opt/tini" | sha256sum -c - \
+    && chmod +x /opt/tini
+
+ARG BITCOIN_VERSION=0.17.0
```

Comment:
Because that's what the other image use. I think this should all be changed at once in a single PR.
Anyway, the only reason for this dependency is to get `bitcoin-cli`, so the version does not matter too much.

Reply:
Ok, thanks